format/Schema.fbs

```diff
@@ -243,23 +243,23 @@ table DictionaryEncoding {
 /// ----------------------------------------------------------------------
 /// A field represents a named column in a record / row batch or child of a
 /// nested type.
-///
-/// - children is only for nested Arrow arrays
-/// - For primitive types, children will have length 0
-/// - nullable should default to true in general
 
 table Field {
   // Name is not required, in i.e. a List
```

Comment:
most of the rest of the class, uses triple slashes "///" could you make that change?

Reply:
Good point, done.